app/javascript/core/axes/bar.js

```diff
@@ -154,6 +154,10 @@ wpd.BarAxes = (function() {
         };
 
         this.name = "Bar";
+
+        if (wpd.appData.isMultipage()) {
```

Comment:
I'd like to keep `core` pieces independent of the higher level `appData` as much as possible. Can we keep `page` out of both `axes` and `dataset`?

Reply:
Sure, I'll work on moving them out